src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const header = document.querySelector('thead');
+const headers = [...header.querySelectorAll('th')];
+const tableBody = document.querySelector('tbody');
+
+header.addEventListener('click', sortTableByHeaderClick);
+
+function sortTableByHeaderClick(ev) {
+  const columnName = ev.target.textContent;
+  const index = headers
+    .findIndex(title => title.textContent === columnName);
```

Comment:
you can use .cellIndex
https://learn.javascript.ru/dom-navigation#dom-navigation-tables